### ISSUE TITLE
POST request with Multipart Form 

### ISSUE THREAD

**Issue by lfpmartin** [usebruno/bruno#3139]
### I have checked the following:

- [ ] I use the newest version of bruno.
- [X] I've searched existing issues and found nothing related to my issue.

### Describe the bug

Error on Bruno while trying to send a POST request with multiparts params:

1st Part is a CSV File
2nd Part is a simple Json

`Error invoking remote method 'send-http-request': Error: TypeError: Cannot read properties of undefined (reading 'value')`

If the request is sent without body, it reaches the server. 
When including any of the params, it breaks.

### .bru file to reproduce the bug

```
meta {
  name: TaxItems
  type: http
  seq: 3
}

post {
  url: https://{{host}}/v1.0/api/Forms/TaxItems
  body: multipartForm
  auth: bearer
}

headers {
  X-Request-Id: 37BAC21C94331EDE9499576224B64C81
}

auth:bearer {
  token: {{access_token_set_by_collection_script}}
}

body:multipart-form {
  fileContent: @file(C:\template 0810.csv)
  requestData: '''
    {
    	"externalReportingEntity": "00000069"
    }
  '''
}
```

### Screenshots/Live demo link

![image](https://github.com/user-attachments/assets/122511b5-bca9-4471-9883-01626ea5e986)

**Comment by Reece-Carruthers** [comment]
Yeah, few of these issues now for different methods. Hoping a fix is passed soon. Temporary solution is to downgrade to previous version.

#3139, #3135, #3129

**Comment by lfpmartin** [comment]
> Yeah, few of these issues now for different methods. Hoping a fix is passed soon. Temporary solution is to downgrade to previous version.
> 
> #3139, #3135, #3129

Sorry... I made a mistake on flagging that _"I use the newest version of bruno"_.
I didn't realize that version 29 was released yesterday. I'm still on 28.

**Comment by mjhcorporate** [comment]
This bug has been fixed in v1.29.1 -- see [this comment](https://github.com/usebruno/bruno/issues/3137#issuecomment-2359341331). Can you please retest?

**Comment by lfpmartin** [comment]
Just updated to v1.29.1 and it works fine!
